lib/grandpa/vote_message.go

```diff
@@ -22,8 +22,8 @@ type networkVoteMessage struct {
 	msg  *VoteMessage
 }
 
-// receiveMessages receives messages from the in channel until the specified condition is met
-func (s *Service) receiveMessages(ctx context.Context) {
+// receiveVoteMessages receives messages from the in channel until the specified condition is met
```

Comment:
nit a bit OOS, but maybe add what that `specified condition` is? :thinking:

Reply:
addressed